### DISCUSSION TITLE
blog/how-to-read-a-classic-foreign-text/

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#54] [Announcements]
# blog/how-to-read-a-classic-foreign-text/

这篇文章不仅分享了阅读经典外文文本的技巧和方法，还深入探讨了读者在这一过程中可能遇到的身体与心理反应，提供了一种全面的学术阅读指导。它适用于希望提高阅读经典外文作品深度和质量的学者或学生，强调了阅读的耐心、反复、思考与写作的重要性。

https://pathos.page/blog/how-to-read-a-classic-foreign-text/

**Comment by savouer1** [comment]
不知道是不是思维方式的差异，我读过的西方著作中，经常在前两章或更多的前面篇幅中进行艰涩的理念论述，而到了中间部分则转成较容易理解的事例论证，最后部分是总结。如果能把中间部分和开头部分换个位置的话，相信对很多人来说更容易理解，而且很多人可能就是被第一部分劝退的。对于这种书，一个好的导读手册很有帮助，梳理出各种线，然后提起框架，再读原著就会轻松很多。
现在有了AI，对于艰涩的的书，很多人只读Ai摘要和提纲，这样就认为读了这本书。另外一部分人可能只读了能读懂的部分，而跳过了所有读不懂的部分。还有你这种人，反复啃下自己能力能够啃下的所有部分。读书的数量和质量，一直是对立的，这和性格及目的都相关。学习导读手册等预热知识、速览、详读、总结及补漏、读不同译本相互印证等，以几倍的时间读一本书；这是一种态度，而不是一种方法。

**Reply by Pathsis** [reply]
你说的没错，许多理论书的前两章，特别是导论，一方面，引入了许多新的概念和命题，另一方面，对全书做了概览式的说明和论证。所以，不耐下心来读开头的部分，很难继续读下去。

我还是建议，如果译本的某一段或某一句读的不顺，不太懂，一般情况下，翻下原文就会明白，一般情况下，就是翻译出了问题。
